source/AI.h

```diff
@@ -76,7 +76,7 @@ template <class Type>
 	static double TurnToward(const Ship &ship, const Point &vector);
 	static bool MoveToPlanet(Ship &ship, Command &command);
 	static bool MoveTo(Ship &ship, Command &command, const Point &targetPosition, const Point &targetVelocity, double radius, double slow);
-	static bool Stop(Ship &ship, Command &command, double maxSpeed = 0.);
+	static bool Stop(Ship &ship, Command &command, double maxSpeed = 0., const Point *direction = nullptr);
```

Comment:
It would be better to just use `Point direction = Point()` here and avoid using pointers. It still has the same effect, because a default-constructed Point returns false if typecast to bool.